modules/xz/metadata.json

```diff
@@ -16,7 +16,8 @@
         "github:tukaani-project/xz"
     ],
     "versions": [
-        "5.4.5"
+        "5.4.5",
+        "5.4.5.bcr.1"
```

Comment:
hmm, i am not sure how i feel about `debian style version` strings. Usually the version of a ruleset, or module in this case, shouldn't be directly tied to the underlying software. But this isn't a ruleset so idk...

Reply:
Not sure, I went with the method in BCR docs which seems to be how most other modules do it, version matches the software and revisions after .bcr, @fmeum thoughts on versioning like this?

Reply:
This looks good to me in this situation: We are tracking an upstream that isn't Bazel-aware, so the Debian style versioning makes sense.